templates/service-template.yml

```diff
@@ -397,11 +397,11 @@ objects:
                 memory: ${MEMORY_LIMIT}
             livenessProbe:
               httpGet:
-                path: /api/maestro
-                port: 8000
+                path: /healthcheck
+                port: 8083
                 scheme: HTTPS
               initialDelaySeconds: 15
```

Comment:
maybe use 20 as a consistent with readinessProbe.

Reply:
set the livenessProbe initialDelaySeconds greater than readinessProbe
use different value for periodSeconds